samples/vvvv/add.v4p

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE PATCH  SYSTEM "http://vvvv.org/versions/vvvv50beta37.dtd" >
```

Comment:
Any chance you could find a sample under a permissive license?

Reply:
Yes, I changed it to the MIT license.

Reply:
Oh, I missed that you were the author. How did you write this sample file? Is it representative of in-the-wild usage? Sample files are used to train the Bayesian classifier so it's important that they be from "real" applications.

Reply:
It is a normal vvvv patch, albeit a very simple one. I added another one that is more the way vvvv is typically used, so that should be enough for the classifier.